src/declarativetabmodel.h

```diff
@@ -107,7 +106,7 @@ public slots:
     virtual void updateThumbPath(int tabId, QString path) = 0;
 
     // This should be replaced by m_activeTabIndex
```

Comment:
Remove the above comment :)

```m_activeTabIndex``` is as good as ```m_activeTabId``` both would occasionally trigger some searching by index or by tabId.